ext/ext.md

```diff
@@ -31,14 +31,16 @@ Content-Type: application/json
 
 {
     "extensions: [
-        "_<ns>/<ext>/<component>",
+        {
+            "name": "_<ns>/<ext>/<component>"
```

Comment:
Above, we have `_ext/discover`, which would imply this should be `_<ext>/<operation>`. Was there a reason to have the extra path component?

Reply:
As per my understanding, extensions are namespaces and components are contained under an extension. Hence the path ```_ns/extension/component```. If the discover API as to be aligned with this model, one proposal could be ```_oci/ext/discover``` @stevvooe what is your guidance here?

Reply:
Per the discussion in the OCI meeting, we are going with `/v2/_<ext>/<operation>` and `/v2/<name>/_<ext>/<operation>`.